app/hub/Views/SideNavigationView/SideNavigationViewContainer.jsx

```diff
@@ -55,6 +56,10 @@ class SideNavigationViewContainer extends Component {
 			{ href: '/products', icon: 'products', text: t('hub_side_navigation_products') }
 		];
 
+		if (IS_CLIQZ) {
+			menuItems = menuItems.filter(item => item.href !== '/rewards');
```

Comment:
@benstrumeyer can we keep the menuItems a constant? You can do this by declaring it something like this:
> 
		const menuItems = [
			{ href: '/', icon: 'home', text: t('hub_side_navigation_home') },
			{ href: '/setup', icon: 'setup', text: t('hub_side_navigation_setup') },
			{ href: '/tutorial', icon: 'tutorial', text: t('hub_side_navigation_tutorial') },
			{ href: '/plus', icon: 'plus', text: t('hub_side_navigation_supporter') },
			...(IS_CLIQZ ? [{ href: '/rewards', icon: 'rewards', text: t('hub_side_navigation_rewards') }] : []),
			...[{ href: '/products', icon: 'products', text: t('hub_side_navigation_products') }]
		];

Reply:
Great idea, fixed.